src/sass/layout/base/_variables.scss

```diff
@@ -9,3 +9,4 @@ $color-grey-light: #707070;
 $color-grey: #444444;
 $color-black: #000000;
 $color-white: #ffffff;
+$color-black-200: #202020;
```

Comment:
hey,

to nie jest kolor czarny, to jest kolor szary;)

Oraz nazwa zmiennej jest dziwna -> moze nazwij go $color-skills-bg